schwifty/bank_registry/manual_fr.json

```diff
@@ -2398,5 +2374,21 @@
         "name": "SWAN",
         "short_name": "SWAN",
         "primary": true
+    },
+    {
+        "country_code": "FR",
+        "primary": true,
+        "bank_code": "AECF",
+        "bic": "AECFFR21",
+        "name": "AMERICAN EXPRESS CARTE - FRANCE S.A.",
+        "short_name": "AMERICAN EXPRESS CARTE"
+    },
+    {
+        "country_code": "FR",
+        "bic": "CMBRFR2B",
+        "bank_code": "CMBR",
+        "name": "CREDIT MUTUEL DE BRETAGNE",
+        "short_name": "CREDIT MUTUEL DE BRETAGNE",
+        "primary": true
     }
```

Comment:
Wouldn't it be easier to read if we kept attributes in a certain order?

Reply:
True dat. But I think it will take a lot of time, and in that case, might as well just copy everything to our yaml files. 😬

Reply:
Yeah, I meant it only for the newly entered banks